pdfly/extract_annotated_pages.py

```diff
@@ -0,0 +1,35 @@
+"""
+Extract only the annotated pages from a PDF.
+
+Q: Why does this help?
+A: https://github.com/py-pdf/pdfly/issues/97
+"""
+
+from pathlib import Path
+from pypdf import PdfReader, PdfWriter
+
+
+# Check if an annotation is manipulable.
+def is_manipulable(annot) -> bool:
+    return annot.get("/Subtype") not in ["/Link"]
+
+
+# Main function.
+def main(input_pdf: Path, output_pdf: Path | None) -> None:
```

Comment:
```suggestion
def main(input_pdf: Path, output_pdf: Optional[Path]) -> None:
```
Seems that the `| None` syntax is not valid with Python 3.8

Reply:
You also need to import `Optional` from `typing` 🙂

Reply:
Oh no. Thanks a lot for the "Never say never".